modules/dnn/src/graph_simplifier.cpp

```diff
@@ -111,7 +111,7 @@ bool Subgraph::match(const Ptr<ImportGraphWrapper>& net, int nodeId,
 
         for (int j = 0; j < inputNodes.size(); ++j)
         {
-            if (nodes[inputNodes[j]].empty())  // Unknown input node type.
+            if (nodes[inputNodes[j]].empty() || nodes[inputNodes[j]] == "Initializer")  // Unknown input node type.
```

Comment:
Can you please add some note in PRs description what is Initializer and from which version of PyTorch it appears?

Reply:
Can you please also add some tests for new functionality?

Reply:
I have added test for **FrozenBatcnNorm**.

Reply:
> I have added test for **FrozenBatcnNorm** and **UpsampleUnfused** with two inputs

Reply:
Can you please check if the Initializer nodes will remain if before export, `model.eval()` is used? It seems to me that model in training state.